examples/universal-with-router/client/index.js

```diff
@@ -0,0 +1,25 @@
+import 'babel-polyfill'
+import React from 'react'
+import { render } from 'react-dom'
+import { Provider } from 'react-redux'
+import { Router, browserHistory } from 'react-router'
+import { syncHistoryWithStore } from 'react-router-redux'
```

Comment:
Sure, I will! but I got my first star from @FarhadG for this example... it's hard for me to delete it right now ;)